Makefile

```diff
@@ -209,10 +209,13 @@ go.sum: go.mod $(GONOGENERATED_SOURCES)
 
 # Build the docker image
 .PHONY: docker-build
-docker-build: dist/harbor-operator_linux_amd64/manager
-	docker build dist/harbor-operator_linux_amd64 \
-		-f Dockerfile \
-		-t "$(IMG)"
+docker-build:
+	docker buildx build \
+    		--platform linux/arm64,linux/amd64 \
+    		--push \
```

Comment:
Why does build now also push?

Reply:
It seems like this is failing CI

Reply:
add `make docker-build-mutil-arch` cmd, reserve `make docker-build`.